### DISCUSSION TITLE
umijs 4  layout side模式下，重写layouts 引发的问题

### DISCUSSION THREAD

**Discussion by zyh0yh** [umijs/umi#10685] [General]
在重写laoyouts 之后，左侧菜单左下角的头像，没办法去除

**Comment by zyh0yh** [comment]
![image](https://user-images.githubusercontent.com/48547668/223083393-26a0a6eb-94ae-4e93-ac13-b163c9c33aac.png)
这是实际的图，我想要的是左下角的头像取消渲染

**Comment by zyh0yh** [comment]
请求帮助

**Comment by fz6m** [comment]
用的是 layout 插件或者 max 吗，本质是 `ProLayout` ，可以看下 [ProLayout](https://procomponents.ant.design/components/layout#prolayout) 的选项。

同时可以参考 `import { RunTimeLayoutConfig } from "@umijs/max";` 这个类型里可以配置的选项，他用于 `src/app.tsx` 里的 `export const layout: RunTimeLayoutConfig` 。

**Reply by zyh0yh** [reply]
这是我在运行时配置的代码

**Reply by zyh0yh** [reply]
![image](https://user-images.githubusercontent.com/48547668/223297689-99bca230-ccbf-428b-8ec1-94a41486d5af.png)

**Reply by fz6m** [reply]
不需要的地方需要配 `false` ，不是 `undefined`
